docs/runbooks/VirtOperatorRESTErrorsHigh.md

```diff
@@ -0,0 +1,20 @@
+# VirtOperatorRESTErrorsHigh 
+
```

Comment:
Hi @yuhaohaoyu, this alert is not mentioned in the PR description.

Reply:
@assafad Thanks, addressed.